tests/ecosystem/deployment/test_deployment.py

```diff
@@ -35,6 +36,14 @@ def test_deployment(pvc_factory, pod_factory):
                     sanity_helpers.health_check()
                     sanity_helpers.delete_resources()
                 config.switch_ctx(restore_ctx_index)
+                if (
+                    config.ENV_DATA["platform"].lower()
+                    in constants.MANAGED_SERVICE_PLATFORMS
+                ):
+                    try:
+                        post_onboarding_verification()
+                    except exceptions.ResourceWrongStatusException as err_msg:
+                        log.warning(err_msg)
```

Comment:
Shouldn't this fail the test instead of just logging warning?

Reply:
thank you, fixed